project.clj

```diff
@@ -12,26 +12,27 @@
   :pedantic? :abort
 
   :dependencies
-  [[org.clojure/clojure "1.10.1" :scope "provided"]
-   [org.bouncycastle/bcpg-jdk15on "1.66"]
-   [org.bouncycastle/bcprov-jdk15on "1.66"]
-   [byte-streams "0.2.4"]]
+  [[org.clojure/clojure "1.10.3" :scope "provided"]
+   [org.bouncycastle/bcpg-jdk15on "1.70"]
+   [org.bouncycastle/bcprov-jdk15on "1.70"]
+   [org.clj-commons/byte-streams "0.2.10"]
+   [org.clojure/tools.logging "1.2.4"]]
```

Comment:
This library doesn't depend on `tools.logging`; curious why you're adding it here?

Reply:
I originally excluded the logging lib from byte-streams (coming from manifold) following the suggestion: 

```
Possibly confusing dependencies found:
[cloverage "RELEASE"] -> [org.clojure/tools.logging "0.5.0"]
 overrides
[org.clj-commons/byte-streams "0.2.10"] -> [manifold "0.1.9"] -> [org.clojure/tools.logging "1.1.0" :exclusions [org.clojure/clojure]]

Consider using these exclusions:
[org.clj-commons/byte-streams "0.2.10" :exclusions [org.clojure/tools.logging]]

Aborting due to :pedantic? :abort
Error encountered performing task 'cloverage' with profile(s): 'base,system,user,provided,dev,coverage'
Suppressed exit
```
and added the logging dependecy to the :tool profile in order to not break the fuzz test.

However I unfortunately noticed too late by the failing build pipeline that this breaks `lein check` and finally fixed this by adding the logging lib to the top-level deps.

I agree the must be a better way to fix this :blush:...

Reply:
Ah, this is because the `lein-cloverage` plugin is injecting `cloverage` as a dependency when the `:coverage` profile is activated. You can just add the logging dependency in the `:coverage` profile and that should fix the problem without changing the root dependencies.

Reply:
Embarrassing, this is actually the first thing I did but due to a stupid typo I thought it didn't work this way... :relieved: